net-snmp/Dockerfile

```diff
@@ -19,6 +19,6 @@ COPY tmpfiles.template service.template manifest.json config.json.template /expo
 RUN mkdir -p /exports/hostfs/etc/{sysconfig,snmp} /exports/hostfs/var/lib/net-snmp/ && \
     cp /etc/sysconfig/snmpd /etc/sysconfig/snmptrapd /exports/hostfs/etc/sysconfig/ && \
     cp /etc/snmp/snmpd.conf /etc/snmp/snmptrapd.conf /exports/hostfs/etc/snmp/ && \
-    echo "# WAIT_FOR_CONFIGURATION=0" >> /exports/hostfs/etc/sysconfig/snmpd
+    echo "NET_SNMP_PRESTART_HOOK='net-snmp-create-v3-user -a password -x encyption-password admin-user'" >> /exports/hostfs/etc/sysconfig/snmpd
```

Comment:
Won't this run on every start of the container?

Is the idea that one can modify ``NET_SNMP_PRESTART_HOOK`` to add new users through restarting the container?